tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/TerraformDestroy.java

```diff
@@ -23,6 +23,7 @@
     tfVars("var"),
     tfVarFiles("var-file"),
     noColor("no-color"),
+    refreshState("refresh"),
```

Comment:
Refresh will not be added to the terraform command. Please add a test for this to TerraformDestroyTest to validate when it is added.

Reply:
Removed refreshState